src/app/components/content-actions/content-actions.component.ts

```diff
@@ -219,16 +217,4 @@ export class ContentActionsComponent {
     );
     return msg;
   }
```

Comment:
showUnenrollButton is not used ?

Reply:
No, its not being used anymore and also isObject empty method too